packages/blockchain-wallet-v4-frontend/src/layouts/Public/components/Footer/ExchangeLogin/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import styled from 'styled-components'
+
+import { Icon, Link, Text } from 'blockchain-info-components'
+
+const ExchangePill = styled.div`
+  cursor: pointer;
```

Comment:
I think cursor is not necessary since it's inside link already

Reply:
I thought the same thing, but if I remove it, it doesn't work...